web/src/filter.ts

```diff
@@ -727,7 +727,7 @@ export class Filter {
         return this.has_operator("search");
     }
 
-    is_non_huddle_pm(): boolean {
+    is_non_direct_message_group_pm(): boolean {
```

Comment:
Let's call this `is_non_group_direct_message`, or maybe there was some other name discussed in Zulip for the concept for a 1:1 direct message.

`pm` is an old reference to "private messages" , and not helpful.

Reply:
I could not find other names discussed on CZO. So I renamed to `is_non_group_direct_message`.